Mathlib/Geometry/Manifold/ChartedSpace.lean

```diff
@@ -250,10 +250,14 @@ theorem StructureGroupoid.locality (G : StructureGroupoid H) {e : PartialHomeomo
   G.locality' e h
 #align structure_groupoid.locality StructureGroupoid.locality
 
-theorem StructureGroupoid.eq_on_source (G : StructureGroupoid H) {e e' : PartialHomeomorph H H}
+theorem StructureGroupoid.mem_of_eqOnSource (G : StructureGroupoid H) {e e' : PartialHomeomorph H H}
     (he : e ∈ G) (h : e' ≈ e) : e' ∈ G :=
-  G.eq_on_source' e e' he h
-#align structure_groupoid.eq_on_source StructureGroupoid.eq_on_source
+  G.mem_of_eqOnSource' e e' he h
+#align structure_groupoid.eq_on_source StructureGroupoid.mem_of_eqOnSource
+
+theorem StructureGroupoid.mem_iff_of_eqOnSource (G : StructureGroupoid H)
+    {e e' : PartialHomeomorph H H} (h : e ≈ e') : e ∈ G ↔ e' ∈ G :=
+  ⟨fun he ↦ G.mem_of_eqOnSource he (Setoid.symm h), fun he' ↦ G.mem_of_eqOnSource he' h⟩
```

Comment:
In addition to the rename, I added this one small lemma, an `iff` version of `mem_of_eqOnSource`. This is to mirror how lemmas like `HasDerivAt.congr_of_eventuallyEq` have an `iff` counterpart `Filter.EventuallyEq.hasDerivAt_iff`. If there is any doubt about this, I'll be happy to remove it from this particular PR.

Reply:
This lemma has been removed from this PR and moved to #9827.